src/main/java/org/kpmp/datasetSummary/DatasetSummary.java

```diff
@@ -0,0 +1,74 @@
+package org.kpmp.datasetSummary;
+
+public class DatasetSummary {
+    private String omicsType;
+    private String dataType;
+    private String dataTypeShort;
+    private Long aki;
```

Comment:
These next couple might be better named akiCount, ckdCount, etc so we know what they are.